test/functional/wallet_labels.py

```diff
@@ -49,7 +49,7 @@ def invalid_label_name_test(self):
         assert_equal(response[0]['error']['message'], "Invalid label name")
 
         for rpc_call in rpc_calls:
-            assert_raises_rpc_error(-11, "Invalid label name", *rpc_call, "*")
+            assert_raises_rpc_error(-11, "Invalid label name", *rpc_call, label="*")
```

Comment:
Note: This named arg here is passed down to the non-wrapped methods of `RPCOverloadWrapper` class as well because those are present in `rpc_calls` list, but _after_ the positional args.